server/conf/evolutions/default/23.sql

```diff
@@ -0,0 +1,23 @@
+
+# --- !Ups
+CREATE TYPE REWARD_TYPE AS ENUM (
+  'PoW',
+  'PoS',
+  'TPoS_OWNER',
+  'TPoS_MERCHANT'
+);
+
+CREATE TABLE block_rewards(
+  blockhash HASH_TYPE NOT NULL,
+  value AMOUNT_TYPE NOT NULL,
+  address ADDRESS_TYPE NOT NULL,
+  type REWARD_TYPE NULL,
+  -- constraints
+  CONSTRAINT block_rewards_pk PRIMARY KEY (blockhash, address)
+);
+
+
+# --- !Downs
+
+DROP TABLE block_rewards;
+DROP TYPE REWARD_ADDRESS_TYPE;
```

Comment:
This type doesn't exists, please tests your down scripts manually.

Usually, before committing an evolution script, you run the ups manually and then, the downs to verify everything is correct.

A note for myself, we should think on adding a test just to verify the evolutions.